### PR TITLE
Documentation typos

### PR DESCRIPTION
**Pull request by kblibr**
# Pull Request Checklist

* [x] Have you read [How to write the perfect pull request](https://github.com/blog/1943-how-to-write-the-perfect-pull-request)?
* [x] Have you read through the [contributor guidelines](https://www.playframework.com/contributing)?
* [x] Have you signed the [Lightbend CLA](https://www.lightbend.com/contribute/cla)?
* [x] Have you referenced any issues you're fixing using [commit message keywords](https://help.github.com/articles/closing-issues-using-keywords/)?
* [x] Have you added copyright headers to new files?
* [x] Have you checked that both Scala and Java APIs are updated?
* [x] Have you updated the documentation for both Scala and Java sections?
* [x] Have you added tests for any changed functionality?

# Helpful things

## Fixes

Fixes #xxxx

## Purpose

What does this PR do?

## Background Context

Why did you take this approach?

## References

Are there any relevant issues / PRs / mailing lists discussions?

### CODE REVIEW

File: documentation/manual/working/scalaGuide/main/json/ScalaJsonHttp.md [hunk 1]
```diff
@@ -115,7 +115,7 @@ HTTP/1.1 400 Bad Request
 Content-Type: application/json
 Content-Length: 79
 
-{"status":"KO","message":{"obj.name":[{"msg":"error.path.missing","args":[]}]}}
+{"status":"OK","message":{"obj.name":[{"msg":"error.path.missing","args":[]}]}}
```
Comment (renatocaval) [inline documentation/manual/working/scalaGuide/main/json/ScalaJsonHttp.md]: That's not the intention. 

The example is showing a response to a `BadRequest`. The `status` field on the json payload should not be confused with the status code. 

I agree that it's a smell to have another `status` field in a REST api and that this leads to confusion. Maybe we should instead remove the `status`.

Comment (renatocaval) [reply documentation/manual/working/scalaGuide/main/json/ScalaJsonHttp.md]: I clicked to fast on "Request change".

To make my point clear, the PR as it is now is should either be closed or we should change the documentation and remove the `status` from that message. We don't need to duplicate a protocol on top of REST. 

@kblib, would you be willing to take on that?

Comment (kblibr) [reply documentation/manual/working/scalaGuide/main/json/ScalaJsonHttp.md]: Happy to remove the status field from the responses.

There was not equivelant documentation in the javaJson to compare against, so I simply assumed it was a typo.  (an easy one to make)

**Comment by renatocaval** [comment]
Thanks @kblibr,

We still have some examples in [ScalaJsonHttpSpec](https://github.com/playframework/playframework/blob/master/documentation/manual/working/scalaGuide/main/json/code/ScalaJsonHttpSpec.scala) using the 'status' when constructing the Json.

This file is used to include code fragments on the page you have edited.

Could you also include them to this PR? Thanks.

**Comment by kblibr** [comment]
Thanks for helping me make a good "comprehensive" pull request.  I am just now learning about play, and came across this in the docs as part of my initial reading.

**Comment by renatocaval** [comment]
> Thanks for helping me make a good "comprehensive" pull request. I am just now learning about play, and came across this in the docs as part of my initial reading.

You are welcome!
Thanks for the patience and the extra effort to get it over the line. 

This PR is approved and will be automatically merged when the build is green.

**Pull request merged**